packages/react-native-sdk/docusaurus/docs/reactnative/04-ui-components/livestream/host-livestream.mdx

```diff
@@ -0,0 +1,116 @@
+---
+id: host-livestream
+title: HostLivestream
+---
+
+import ImageShowcase from '@site/src/components/ImageShowcase';
+import HostLivestreamStart from '../../assets/04-ui-components/livestream/host-livestream-start.png';
+import HostLivestreamEnd from '../../assets/04-ui-components/livestream/host-livestream-end.png';
+
+import HostLivestreamTopView from '../../common-content/ui-components/livestream/host-livestream/host-livestream-top-view.mdx';
+import LivestreamLayout from '../../common-content/ui-components/livestream/livestream-layout.mdx';
+import HostLivestreamControls from '../../common-content/ui-components/livestream/host-livestream/host-livestream-controls.mdx';
+import DurationBadge from '../../common-content/ui-components/livestream/duration-badge.mdx';
+import FollowerCount from '../../common-content/ui-components/livestream/follower-count.mdx';
+import LiveIndicator from '../../common-content/ui-components/livestream/live-indicator.mdx';
+import HostStartStreamButton from '../../common-content/ui-components/livestream/host-livestream/host-start-stream-button.mdx';
+import LiveStreamMediaControls from '../../common-content/ui-components/livestream/livestream-media-controls.mdx';
+import OnStartStreamHandler from '../../common-content/ui-components/livestream/host-livestream/on-start-stream-handler.mdx';
+import OnEndStreamHandler from '../../common-content/ui-components/livestream/host-livestream/on-end-stream-handler.mdx';
+
+The `HostLivestream` is the UI component that allows you to show the live stream view in the app on the host's end with minimal efforts. So you don't need to take care much about each feature that you need to build a video live stream call screen with this component.
+
+Basically what you can do with the `HostLivestream` is:
+
+- A complete live stream UI with ability to stream host's video.
+- Duration of the live stream with live indicator.
+- Whether the stream is live and number of participant's watching the stream.
+- Media controls to control publish and un-publish stream.
+
+<ImageShowcase
+  items={[
+    {
+      image: HostLivestreamStart,
+      caption: 'Host Live Stream Start',
+      alt: 'Host Live Stream Start',
+    },
+    {
+      image: HostLivestreamEnd,
+      caption: 'Host Live Stream End',
+      alt: 'Host Live Stream End',
+    },
+  ]}
+/>
+
+## General usage
+
+Let's see how to show the `HostLivestream` UI:
+
+```tsx {13}
```

Comment:
```
tsx {13}
```

instead of 13 can you put `// highlight-next-line` instead.. its the recommended solution from docusaurus for maintainability

Reply:
Umm, I was asked to use this convention in the past by @vishalnarkhede hence I used this. Not sure which one is better. The other also has a higher LOC effect on code.

Reply:
Anyways done ✅